Makefile

```diff
@@ -35,6 +35,7 @@ lint:
 	  -- index.js
 	$(ESLINT) \
 	  --env node \
+	  --env es6 \
 	  --env mocha \
```

Comment:
any preference to where the `--env es3` should go in the `-- index.js` command? :)